Woodoku/src/entity/woodoku_board.py

```diff
@@ -1,12 +1,11 @@
-from typing import Dict, List, Tuple, Set, Iterable
+from typing import List, Tuple, Set, Iterable
 
 import numpy as np
 from numpy import ndarray
 
 from ui.utils import *
```

Comment:
```suggestion
from ui.utils import get_input, red, orange
```
only import what is necessary